configure.ac

```diff
@@ -387,6 +387,13 @@ esac
 have_libftd2xx="no"
 AM_CONDITIONAL([HAVE_LIBFTD2XX], [test "x$have_libftd2xx" = xyes])
 
+# libsystemd
+AC_SEARCH_LIBS([sd_notify], [systemd], [have_libsystemd="yes"])
```

Comment:
If you see have_libftd2xx above, you should really initialise it to no before you use it here.

Reply:
No problem. I'll follow the examples from the above location. 

What do you think about making linking against `libsystemd` a command line option to the build system? I'm thinking build systems that have `libsystemd` may silently cause built binaries to link against `libsystemd`, and this could lead to a bit of a dependency problem, unless they're aware of this new change.

Reply:
Yes, it may make sense, see e.g. https://github.com/OpenLightingProject/ola/blob/master/configure.ac#L494-L518 .

I was wondering, but at least it's only libsystemd, it doesn't require systemd itself.